modules/ROOT/pages/creating-a-custom-policy.adoc

```diff
@@ -0,0 +1,292 @@
+        ]
+      }
+    }
+  },
+  "required": [
+    "username",
+    "password"
+  ],
+  "unevaluatedProperties": false,
+  "@context": {
+    "@vocab": "anypoint://vocabulary/policy.yaml#",
+    "security": "anypoint://vocabulary/policy.yaml#"
+  },
+  "$id": "basic-authentication-simple",
+  "$schema": "https://json-schema.org/draft/2019-09/schema"
+}
+----
+
+=== Create the Definition Metadata YAML File
+
+The YAML file provides information about the policy and its requirements. The YAML file is not tied to any specific runtime or binary. 
+
+The file must have these fields configured:
+
+* Header
++
+Header must be #%Policy Definition 0.1.
+* Name
++
+User friendly name that is used for displaying the policy name in API Manager UI. Must be a string value.
+* Description
++
+Must be a string value.
+
+* Category
++
+Category of which the policy belongs. Used to group and filter policies in API Manager UI, any string value is valid. For example, security, quality of service, compliance, troubleshooting, and transformation.
+
+You can view this xref:https://github.com/mulesoft/ang-containers/blob/master/data/management/dialects/policy-definition.yaml[policy definition file] in GitHub with more information about all the supported fields. Some of the fields are auto-completed by the platform during the publication process, so not every field is read or is necessary in the file.
+
+The following code is an example of a basic authentication definition YAML file:
+----
+##%Policy Definition 0.1
+name: Basic authentication - Simple
+description: Enforces HTTP Basic authentication according to the details configured in the policy.
+category: Security
+providedCharacteristics:
+  - Requires authentication
+requiredCharacteristics: []
+interfaceScope: ["api",
+----
+
+[[create-implementation-files]]
+== Create Implementation Files
+The policy implementation files that are required to make the policy available in a runtime are a binary file and a metadata file. The binary file contains the policy logic and manages the runtime execution. The file can be a WebAssembly (WASM) for Anypoint Flex Gateway or a JAR file for Mule 4 runtime. The metadata file describes what technology the binary file applies to and from which version.
+
+=== Create Metadata YAML File
+The metadata YAML file provides details about the specific implementation of the policy definition. A single policy definition may several implementations, each an independent asset in Exchange or different versions of the same Exchange asset. 
+
+The metada YAML file must include the following fields:
+
+* Header
++
+#%Policy Implementation 1.0
+* Technology
++
+Either `mule4`` or `flexGateway`.
+* Name
+* minRuntimeVersion
++
+A semver version representing the first runtime version that the implementation binary is compatible with. For example, a mule binary might only be compatible from 4.3.0 onwards.
+
+The YAML file can also include:
+
+* Release notes
+* maxRuntimeVersion
```

Comment:
I think Mule Runtime does not support maxRuntimeVersion for policies, therefore it might be better not to include this in the documentation, at least for the time being

Reply:
fixed